packages/libs/eda/src/lib/map/analysis/MapLegend.tsx

```diff
@@ -1,33 +1,49 @@
-import React from 'react';
+import Spinner from '@veupathdb/components/lib/components/Spinner';
 import PlotLegend from '@veupathdb/components/lib/components/plotControls/PlotLegend';
 import { LegendItemsProps } from '@veupathdb/components/lib/components/plotControls/PlotListLegend';
 
 interface Props {
   legendItems: LegendItemsProps[];
+  isLoading: boolean;
   title?: string;
   showCheckbox?: boolean;
 }
 
 export function MapLegend(props: Props) {
-  const { legendItems, title = 'Legend', showCheckbox } = props;
+  const { legendItems, isLoading, title = 'Legend', showCheckbox } = props;
+  console.log(
+    '🚀 ~ file: MapLegend.tsx:14 ~ MapLegend ~ isLoading:',
+    isLoading
+  );
+  console.log(
```

Comment:
Remember to remove these guys :-)

Reply:
Oh dear! I put those in there because I thought I broke something and was confuzzled! But it was something else (#255).

Removed in: https://github.com/VEuPathDB/web-monorepo/pull/232/commits/d14593eb0f00c6faf06af461f100d9ea57ff9444